cmake/UserverTestsuite.cmake

```diff
@@ -59,6 +59,12 @@ function(userver_venv_setup)
   set(venv_bin_dir "${venv_dir}/bin")
   set("${python_output_var}" "${venv_bin_dir}/python" PARENT_SCOPE)
 
+  if(USERVER_PIP_USE_SYSTEM_PACKAGES)
+    list(APPEND ARG_VIRTUALENV_ARGS "--system-site-packages")
+  endif()
+
+  list(APPEND ARG_PIP_ARGS ${USERVER_PIP_OPTIONS})
```

Comment:
In Conan builds, `UserverTestsuite.cmake` can be included as a standalone file, even without including userver itself. This code should behave well if `USERVER_PIP_OPTIONS` is undefined. Let's wrap this line in an `if(USERVER_PIP_OPTIONS)` just in case